Documentation_displayed_on_the_portal/Share_Dataset_Instruction_Page.md

```diff
@@ -123,10 +123,10 @@ Add the [required](#required) files to the root directory of the dataset and con
 
 All files in the dataset must be added to the repository using one of the two commands below. Copying content from another location into your local copy of the repository without using those commands **will not work**.
 
-- For data files on FTP or HTTP servers, use the `web` remote to populate the data:
+- For data files on FTP or HTTP servers, use the `web` remote to populate the data: for each individual file:
```

Comment:
```suggestion
- For data files on FTP or HTTP servers, use the `web` remote to populate the data for each individual file:
```

Reply:
That colon was intentional, but I can see it working either way, so if you prefer without, I am happy to do that.